howso/direct/client.py

```diff
@@ -413,6 +414,8 @@ def create_trainee(  # noqa: C901
         resources : howso.openapi.models.TraineeResources or dict, optional
             (Not implemented) Customize the resources provisioned for the
             Trainee instance.
+        trainee_id : str, optional
```

Comment:
If `id` is already available on `trainee` why do we need a separate parameter?

Reply:
good question, should i just get rid of this for now until v3 / we separate names and ids altogether?